bot/resources/evergreen/trivia_quiz.json

```diff
@@ -269,5 +287,322 @@
+      "img_url": "https://wikimedia.org/api/rest_v1/media/math/render/png/d22b8566e8187542966e8d166e72e93746a1a6fc",
+      "answer": "Heron's formula"
+    },
+    {
+      "id": 209,
+      "dynamic_id": 201,
+      "question": "Solve the following system of linear equations (format your answer like this <x> & <y>):\n{}x + {}y = {},\n{}x + {}y = {}",
+      "answer": "{} & {}"
+    },
+    {
+      "id": 210,
+      "dynamic_id": 202,
+      "question": "What's {} + {} mod {} congruent to?",
+      "answer": "{}"
+    },
+    {
+      "id": 211,
+      "question": "What is the bottom number on a fraction called?",
+      "answer": "denominator"
+    },
+    {
+      "id": 212,
+      "dynamic_id": 203,
+      "question": "How many vertices are on a {}gonal prism?",
+      "answer": "{}"
+    },
+    {
+      "id": 213,
+      "question": "What is the term used to describe two triangles that have equal corresponding sides and angle measures?",
+      "answer": "congruent"
+    },
+    {
+      "id": 214,
+      "question": "⅓πr2h is the volume of which 3 dimensional figure?",
+      "answer": "cone"
+    },
+    {
+      "id": 215,
+      "dynamic_id": 204,
+      "question": "Find the square root of -{}.",
+      "answer": "{}i"
+    },
+    {
+      "id": 216,
+      "question": "In set builder notation, {p/q | q ≠ 0, p & q ∈ Z} represents what?",
+      "answer": "Rational Numbers"
+    },
+    {
+      "id": 217,
+      "question": "What is the natural log of -1 (use i for imaginary number)?",
+      "answer": "pi*i, pii, πi"
+    },
+    {
+      "id": 218,
+      "question": "When is the *inaugural* World Maths Day (format your answer in MM/DD)?",
+      "answer": "03/13"
+    },
+    {
+      "id": 219,
+      "question": "As the Fibonacci sequence extends to infinity, what's the ratio of each number `n` and its preceding number `n-1` approaching?",
+      "answer": "Golden Ratio"
+    },
+    {
+      "id": 220,
+      "question": "0, 1, 1, 2, 3, 5, 8, 13, 21, 34 are numbers of which sequence?",
+      "answer": "Fibonacci"
+    },
+    {
+      "id": 221,
+      "question": "Prime numbers only have __ factors.",
+      "answer": "2, two"
+    },
+    {
+      "id": 222,
+      "question": "In probability, the ________ ______ of an experiment or random trial is the set of all possible outcomes of it.",
+      "answer": "sample space"
+    },
+    {
+      "id": 223,
+      "question": "In statistics, what does this formula represent?",
+      "img_url": "https://www.statisticshowto.com/wp-content/uploads/2013/11/sample-standard-deviation.jpg",
+      "answer": "sample standard deviation, standard deviation of a sample"
+    },
+    {
+      "id": 224,
+      "question": "\"Hexakosioihexekontahexaphobia\" is the fear of which number?",
+      "answer": "666"
+    },
+    {
+      "id": 225,
+      "question": "Prove or disprove the Riemann Hypothesis",
+      "answer": ""
```

Comment:
Are we missing an answer here?

Reply:
Hmm that's a bit problematic as this question can't be solved

Reply:
well it's not going to harm the game, there's no correct answer so no one will get the points. just a fun little easter egg.

Reply:
I don't think it is fun but more frustrating than anything to have a question that doesn't have an answer. Can you please remove it?

Reply:
ah... ok, i understand.

Reply:
changes implemented, resolved :D